tests/integration/migrations/extract-person-contacts.js

```diff
@@ -0,0 +1,66 @@
+var utils = require('./utils');
+
+var ANY_STRING = new RegExp('^.*$');
+var ANY_NUMBER = new RegExp('^[0-9]*(\\.[0-9]*)?$');
```

Comment:
Not important based on how it's being used I don't think, but FWIW that means `.` and an empty string is a number. Nit picking, I know :-)